src/index.js

```diff
@@ -117,21 +103,14 @@ function main(client, options) {
 	});
 }
 
-function EvolvAssetManager(options) {
-	const client = new Evolv(options);
+function EvolvAssetManager(client) {
+	client.context.set('web.url', window.location.href);
 
 	// Expose client and context proprties
 	Object.defineProperty(this, 'client', { get: function () { return client }});
```

Comment:
Do we still need to do this now that we're attaching the client to the window?

Reply:
Yeah, the client shouldn't collect any attributes, the AssetManager should pass what it believes is relevant.